app/src/gui/components/notebook/index.tsx

```diff
@@ -138,6 +139,7 @@ export default function NotebookComponent({project}: NotebookComponentProps) {
   const theme = useTheme();
   const mq_above_md = useMediaQuery(theme.breakpoints.up('md'));
   const history = useNavigate();
+  const [recordLabel, setRecordLabel] = useState('RRecord');
```

Comment:
RRecord?

Reply:
Ah sorry, da,n
this was a test forgot to remove this. Thanks for picking.